KsApi/models/lenses/RewardLenses.swift

```diff
@@ -24,7 +24,8 @@ extension Reward {
         shippingRulesExpanded: $1.shippingRulesExpanded,
```

Comment:
I'm still formulating some thoughts about how we can replace the `Foo.template.lens |> ...` pattern, so I used this as an opportunity to actually get into the guts of one to learn more about it.